### PR TITLE
User-Details

### PR DESCRIPTION
**Pull request by shrey27**
Added Features
- Update User details from settings page
- Refactored db and related services
- Suggested options updated
- Find followers, following list from modal on profile page
- Private routes added

### CODE REVIEW

File: src/frontend/routes/index.js [hunk 1]
```diff
@@ -46,17 +49,20 @@ export const NOTFOUND = '*';
...
+
+    <Route exact path={LANDING} element={<PrivateRoute />}>
+      <Route exact path={HOMEPAGE} element={<Homepage />} />
+      <Route exct path={POST} element={<IndividualPost />} />
+      <Route exct path={EXPLORE} element={<ExploreFeed />} />
+      <Route exct path={BOOKMARK} element={<Bookmark />} />
+      <Route exct path={NOTIFICATION} element={<Notification />} />
+      <Route exct path={PROFILE} element={<Profile />} />
+      <Route exct path={USERPROFILE} element={<AnyUserProfile />} />
+      <Route exct path={SETTINGS} element={<Settings />} />
```
Comment (romabulani) [inline src/frontend/routes/index.js]: I think there is typo in `exact`. Check this once

**Review by romabulani — APPROVED** [review]
- Clean and modular code
- Functionality is working as expected
- Good to merge

File: src/frontend/routes/index.js (hunk 1 above)
Comment (shrey27) [reply src/frontend/routes/index.js]: thanks for pointing that out

**Review by lomesshh — APPROVED** [review]
Code looks clean and modular
All functionality are working as expected 
Looks good to merge

**Pull request merged**